### DISCUSSION TITLE
hao to delete not used rs

### DISCUSSION THREAD

**Discussion by warfu** [argoproj/argo-cd#5113] [Q&A] [answered]
# Summary
hao to delete not used rs

![image](https://user-images.githubusercontent.com/1465688/102952362-f3120a00-4509-11eb-8a60-cd805905b4dd.png)

 

# Motivation

 

# Proposal

**Comment by servo1x** [comment, marked as answer]
Please see: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#clean-up-policy
